### DISCUSSION TITLE
Accessing different Motors while driving EV3

### DISCUSSION THREAD

**Discussion by g-qwz** [pybricks/support#1205] [Q&A] [answered]
Hey, is there any way to access other Motors (e.g. on Port 3) when using the DriveBase to drive (left and right Motors are on Port 1 & 2)

**Comment by laurensvalk** [comment]
Do you mean at the same time?

Perhaps you can make use the `drive()` function which only starts driving and won't block your program from doing something else in the mean time.

**Comment by g-qwz** [comment]
Yes, I meant at the same time, sorry for the confusion

I am going to try using the drive() method, but just out of curiosity: Is there an implementation for doing the things in parallel e.g. threading?

**Reply by laurensvalk** [reply, marked as answer]
Newer versions of Pybricks support async/await to do multitasking, but we don't have a recent release for EV3 unfortunately.
